deploy/FlapperInit.sol

```diff
@@ -106,4 +133,28 @@ library FlapperInit {
         PipLike(OracleWrapperLike(wrapper).pip()).kiss(wrapper);
         dss.chainlog.setAddress(clKey, wrapper);
     }
+
+    function initSplitter(        
+        DssInstance memory dss,
+        address splitter,
+        SplitterConfig memory cfg
+    ) internal {
+        // Sanity checks
+        SplitterLike splitter_ = SplitterLike(splitter);
+        require(splitter_.vat()     == address(dss.vat),     "Splitter vat mismatch");
+        require(splitter_.daiJoin() == address(dss.daiJoin), "Splitter daiJoin mismatch");
+
```

Comment:
This is something that we need to decide our policy about but I'm wondering if we should add a sanity check that `splitter_.farm()` matches a `cfg.farm` that we get as a paramter.

These immutables deployer-integrity checks are pretty annoying but we got a similar issue once (about not validating `univ3Factory`). Here allegedly a malicious farm can still the 30% of streamed dai.

And while we're at it if we do that we should probably add similar checks for the flapper `pair` and `receiver` in `initFlapperUniV2`.

Reply:
Ok I guess it doesn't hurt to add those. Done.